### ISSUE TITLE
Train on batch

### ISSUE THREAD

**Issue by jahdiel** [yhenon/keras-frcnn#83]
It seems that the training is not done in batches instead it is doing online training (updating the weights for each image), because the data augmentation only returns one image. I would like to be able to train various images before updating the loss. Has anyone been able to achieve this?

**Comment by yuayi521** [comment]
you can use fit_generator function

**Comment by thealchemist89** [comment]
Hi @yuayi521,

Do you know how we could perform data augmentation (ImageDataGenerator) such that it also takes into account the annotated bounding boxes for localization purposes not just classification? I have some some queries on how we could achieve this while training using fit_generator function subsequently as well.
